helpers/loanHelpers/userCollection.tsx

```diff
@@ -342,7 +176,7 @@ const setObligations = async (
         highestBid: obligation.highest_bid
       };
     })
-    .filter((obl: any) => obl.debt !== 0);
+    .filter((obl: any) => obl.debt.toString() != 0);
```

Comment:
@bowtiedpyro verify if this function can be removed (as discussed)

Reply:
This formats our obligations as presented in the FED, for now we need this. Will bring it up with SDK dev to validate if we want to ship this logic to the SDK